web/src/main/java/com/softserve/osbb/controller/HouseController.java

```diff
@@ -248,21 +263,25 @@ public boolean isApartmentNumberValid (@PathVariable("houseId") Integer houseId,
             logger.error("error finding house by id: ", houseId);
             throw new HouseNotFoundException(e);
         }
+        
         return new ResponseEntity<>(houseResource, HttpStatus.OK);
     }
 
     @RequestMapping(value = "/{houseId}", method = RequestMethod.DELETE)
-    public ResponseEntity deleteHouseById(@PathVariable("houseId") Integer houseId) {
+    public ResponseEntity<?> deleteHouseById(@PathVariable("houseId") Integer houseId) {
```

Comment:
It was a raw type, added wildcard just to avoid compiler complains. Generic could be a House, but...
This method has anoter one problem - allways returns OK... It can be rewritten (in a scope of separate task). The aim was just co clean up.